examples/scripts/sparsecode_poisson_pgm.py

```diff
@@ -146,10 +196,10 @@ def plot_results(hist, str_ss, L0, xsol, xgt, Amat):
 
 """
 Use default PGMStepSize object, set L0 based on norm of Forward
-operator and set up AcceleratedPGM solver object. Run the solver and
+operator and set up AcceleratedPGM solver object.  Run the solver and
 plot the recontructed signal and convergence statistics.
 """
-L0 = snp.linalg.norm(D, 2) ** 2  # initial reciprocal of gradient descent step size
+L0 = (snp.linalg.norm(D0, 2) + snp.linalg.norm(D1, 2)) ** 2
```

Comment:
Is the square of the sum of the norms really what you want here?

Reply:
Updated this in f7187e3